### PR TITLE
implement the pipeline export function

### PR DESCRIPTION
**Pull request by maye-msft**
Add the Javascript function to export the pipeline as file in JSON format. the file can be imported as well.

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -1671,6 +1671,25 @@ <h1 class="modal-title fs-5" id="previewModalLabel">Pipeline Preview</h1>
                     }
                     that.currentPipelineObject['serving'].push(newTask)
                     that.currentPipelineServingTask = that.currentPipelineObject['serving'][that.currentPipelineObject['serving'].length - 1]
+                },
+                exportPipeline() {
+                    var that = this
```
Comment (siliang-j-1225) [inline web/index.html]: is `that` used?

File: web/index.html [hunk 2]
```diff
@@ -1671,6 +1671,25 @@ <h1 class="modal-title fs-5" id="previewModalLabel">Pipeline Preview</h1>
...
+                    var pipelineName = this.currentPipelineName
+                    var pipelineObject = this.currentPipelineObject
+                    var pipelineJson = JSON.stringify(pipelineObject, null, 2)
+                    var blob = new Blob([pipelineJson], {
+                        type: "application/json"
+                    });
+
+                    var tempLink = document.createElement("a");
+                    let textArea = document.querySelector("textarea");
+                    var taBlob = new Blob([textArea.value], {type: 'text/plain'});
```
Comment (siliang-j-1225) [inline web/index.html]: what is `taBlob` here?

File: web/index.html [hunk 3]
```diff
@@ -1671,6 +1671,25 @@ <h1 class="modal-title fs-5" id="previewModalLabel">Pipeline Preview</h1>
...
+                    
+                    tempLink.setAttribute('href', URL.createObjectURL(blob));
```
Comment (thurstonchen) [inline web/index.html:1688]: Would it be URL.createObjectURL(**taBlob**) here?

File: web/index.html (hunk 2 above)
Comment (maye-msft) [reply web/index.html]: @siliang-j-1225 it is fixed. Thanks for finding the issue.

File: web/index.html (hunk 1 above)
Comment (maye-msft) [reply web/index.html]: @siliang-j-1225  it is fixed. Thanks for finding this issue.

**Review by siliang-j-1225 — APPROVED** [review]
LGTM. thanks for fixing.

**Pull request merged**